skyportal/models/telescope.py

```diff
@@ -63,14 +64,33 @@ def observer(self):
         try:
             return self._observer
         except AttributeError:
+            if (
```

Comment:
```
try:
    lon = float(self.lon)
    lat = float(self.lat)
    if np.isnan(lon) or np.isnan(lat):
        return None
except TypeError:
    return None
```

Reply:
Anyway, that was the suggestion I was going to make; exception handling version is fine too.